Variables/Arithmetic operators/tests/test_task.py

```diff
@@ -0,0 +1,116 @@
+import importlib
+from io import StringIO
+import sys
+import unittest
+from unittest.mock import patch
+
+
+def try_import():
+    import arithmetic_operators
+    return arithmetic_operators
+
+
+class TestCase(unittest.TestCase):
+    task_name = 'arithmetic_operators'
+
+    def setUp(self):
+        try:
+            # Stores output from print() in actualOutput
+            with patch('sys.stdout', new=StringIO()) as self.actualOutput:
+                # Loads submission on first test, reloads on subsequent tests
+                if self.task_name in sys.modules:
+                    importlib.reload(sys.modules[self.task_name])
+                else:
+                    importlib.import_module(self.task_name)
+        except NameError as ne:
+            pass
+        except Exception as e:
+            self.fail("There was a problem while loading the solution - {0}. Check the solution for "
+                      "IDE-highlighted  errors and warnings.".format(str(e)))
+
+    def test_addition(self):
```

Comment:
I believe this test is run first (they seem to run in alphabetic order of the test names), and operations in the task are connected so if you make a mistake in any of the first four placeholders you receive the same error -  '10.0 != 20.0 : Something does not ADD up.'

Reply:
Yes, they do run in alphabetic order (this could be changed, but here it will do). I've added test names that are easier to order for the system, but we will need to change the task design in the future, this one is to fragile.